src/test/java/com/qaprosoft/carina/demo/mfp/MobileTest.java

```diff
@@ -148,4 +148,30 @@ public void customDashboardTest() {
         softAssert.assertAll();
     }
 
+    @Test
+    @MethodOwner(owner = "Hostiuk")
+    @TestRailCases(testCasesId = "6")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void customSummaryNutrientsTest() {
+        LoginPageBase loginPage = initPage(getDriver(), LoginPageBase.class);
+        loginPage.login(R.TESTDATA.get("email"), R.TESTDATA.get("password"));
+
+        CommonPageBase commonPage = initPage(getDriver(), CommonPageBase.class);
+        DiaryPageBase diaryPage = (DiaryPageBase) commonPage.clickBottomNavigatorButton(BottomNavigatorButtons.DIARY);
+        CustomSummaryPageBase customSummaryPage = (CustomSummaryPageBase) diaryPage.selectCustomDashboard(
+                CustomDashboardPageBase.Options.CUSTOM);
+        Assert.assertTrue(commonPage.isItemByTextPresent(IConstants.THREE_OF_THREE_NUTRIENTS_SELECTED),
+                IConstants.THREE_OF_THREE_NUTRIENTS_SELECTED + " text isn't present");
+        Assert.assertTrue(customSummaryPage.isDoneButtonActive(), "Save button isn't active");
+        customSummaryPage.uncheckNutrient(CustomSummaryPageBase.Nutrients.FAT);
+        Assert.assertTrue(commonPage.isItemByTextPresent(IConstants.TWO_OF_THREE_NUTRIENTS_SELECTED),
+                IConstants.TWO_OF_THREE_NUTRIENTS_SELECTED + " text isn't present");
+        Assert.assertFalse(customSummaryPage.isDoneButtonActive(), "Save button is active");
+        customSummaryPage.checkNutrient(CustomSummaryPageBase.Nutrients.FAT);
+        customSummaryPage.checkNutrient(CustomSummaryPageBase.Nutrients.SAT_FAT);
+        Assert.assertTrue(commonPage.isItemByTextPresent(IConstants.FOUR_NUTRIENTS_SELECTED),
+                IConstants.FOUR_NUTRIENTS_SELECTED + " text isn't present");
+        Assert.assertFalse(customSummaryPage.isDoneButtonActive(), "Save button is active");
```

Comment:
very good! i like that you put messages to the constants, the only correction - message `"Save button is active"` isn't completed, you should provide more info `Save button is active after choosing 2 nutrients`